src/async/services/socketRe.js

```diff
@@ -0,0 +1,7 @@
+import { get } from "../api";
+
+const apiUri = process.env.REACT_APP_API_URL;
+const socketIoEndpoint = async(credentials) => {
+    return await get(`${credentials}&limit=25&offset=0&rating=g&lang=en`, {});
```

Comment:
Good catch but you don't need to change socketIoEndpoint you need to create another function, something loginEnpoint() without modify the socketIoEndpoint